docs/blog/rendered/campaign-finance.ipynb

```diff
@@ -0,0 +1,1813 @@
+{
```

Comment:
<p>Looks like the plots aren't showing up for whatever reason :(</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ibis-project/ibis/pull/5727/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='49'/>

Reply:
<p>oops I was playing around with the renderers and I accidentally have it swapped to .svg. Fixing now.</p>

Reply:
v5 pushed